photon-lib/src/main/java/org/photonvision/RobotPoseEstimator.java

```diff
@@ -178,41 +182,31 @@ public void setLastPose(Pose2d lastPose) {
      *     pipeline results used to create the estimate
      */
     public Optional<EstimatedRobotPose> update() {
-        if (cameras.isEmpty()) {
-            DriverStation.reportError("[RobotPoseEstimator] Missing any camera!", false);
+        if (camera == null) {
+            DriverStation.reportError("[RobotPoseEstimator] Missing camera!", false);
             return Optional.empty();
         }
 
-        ArrayList<CameraResult> cameraResults = new ArrayList<>(cameras.size());
-        boolean hasTargets = false;
-
-        for (Pair<PhotonCamera, Transform3d> p : cameras) {
-            PhotonPipelineResult pipelineResults = p.getFirst().getLatestResult();
-
-            if (pipelineResults.hasTargets()) hasTargets = true;
-
-            cameraResults.add(new CameraResult(p.getFirst(), p.getSecond(), pipelineResults));
-        }
-
-        if (!hasTargets) {
+        PhotonPipelineResult cameraResult = camera.getLatestResult();
```

Comment:
We should probably be checking to make sure that this result is not the same as the last time we ran the pose estimator, right?

Reply:
It would be a good optimization, but not strictly necessary. If you call `.update()` twice with the same pipeline result, you'll get the same estimate both times. The two estimates will have the same pose _and timestamp_.

Before this PR, where the timestamp was estimated from the latency and current time, you'd end up with a different and less accurate timestamp the second time.

Reply:
Got it! That makes sense. The new wpilib side *DrivePoseEstimator lets you feed in multiple targets from the same timestamp and it'll average them all so technically this is isn't ideal, but it's fine for now